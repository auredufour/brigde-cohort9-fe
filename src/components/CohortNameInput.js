import React from "react";

import {
  InputField,
  InputLabel
} from "./CohortNameInputStyled";


const CohortNameInput = (props) => {
  return (
    <div>
      <InputLabel>
        Cohort Name:
        <InputField
          type="text"
          value={props.value}
          onChange={props.handleChange}
          maxLength="64"
          placeholder="Enter Cohort Name"
          required
        />
      </InputLabel>
    </div>
  );
};

export default CohortNameInput;
