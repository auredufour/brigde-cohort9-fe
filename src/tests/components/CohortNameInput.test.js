import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import CohortNameInput from "../../components/CohortNameInput";

describe("CohortNameInput", () => {
  test("Should render properly", () => {
    const renderer = new ShallowRenderer();
    const element = renderer.render(
      <CohortNameInput />
    );
    expect(element).toMatchSnapshot();
  });
});
