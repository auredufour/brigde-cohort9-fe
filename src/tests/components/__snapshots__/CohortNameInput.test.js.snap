// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CohortNameInput Should render properly 1`] = `
<div>
  <ForwardRef(styled.label)>
    Cohort Name:
    <ForwardRef(styled.input)
      maxLength="64"
      placeholder="Enter Cohort Name"
      required={true}
      type="text"
    />
  </ForwardRef(styled.label)>
</div>
`;
